RULES_AND_GUIDELINES.md

```diff
@@ -0,0 +1,220 @@
+
+## <a name="accumulation-and disposal of trash"></a>Accumulation and Disposal of Trash 
+
+Trash, Recyclables, Compostables and large or hazardous items must be properly disposed of, in a timely manner, in order to protect the public health, safety and welfare of the community. Proper disposal prevents conditions that may create fire, health or safety hazards; and does not harbor pests or impair the aesthetic appearance of the community. General storage of items not designed or intended for outdoor use could constitute a violation. Items may include but are not limited to mattresses; discarded furniture and appliances; car parts and tires; wood pallets; and piled yard or food waste. However, every property and situation is unique and may be evaluated on a case-by-case basis. (As defined by Boulder Code Enforcement)
+
+[^ back to top](#)
+
+## <a name="individual-property"></a>Individual Property
+
+Individual Property (Proposed)
+Interior space, including all drywall and finishings within each unit are under the individual control of the unit owner. However, all structural components and building features (including but not limited to studs, joists and internal construction of the building) are common property and cannot be accessed or changed by an individual or within an individual property.
+
+[^ back to top](#)
+
+## <a name="common-property"></a>Common Property
+
+Common Property (Proposed)
+All areas of the property that are commonly owned and jointly contribute to the property value of the association as a whole. This includes but is not limited to all parking lots, landscaped areas, walkways, fire access, dumpster areas and all exterior and structural components of all buildings. Owners and residents are not permitted to make any changes to Common Property as they would affect all owners in the association. Some changes may be considered if presented to the Board through an Architectural Review Request (e.g., changing paver stones, repairing concrete). Residents may not change any landscaping with the addition or removal of any plants, stones or landscaping materials. Around each building is a 2’ stone perimeter, to meet building codes, prevent pest infestations, avoid damage to buildings and siding and drain water away from building crawlspaces and foundations. These stones cannot be removed or replaced. However, residents are permitted to donate above ground landscape features, that are aesthetically appropriate, do not cause any damage and are easily removable (e.g., garden bench, terracotta or glazed planters, plant stands, sealed bottom, raised bed planters...). Note that any items placed in a common space will immediately be considered a donation to the association. All donations can be removed at any time, without notice, if any board member assesses the item to be inappropriate in aesthetic nature or placement. Some areas of common property are designated for limited use so check guidelines for ‘limited common property’.
```

Comment:
`However, residents are permitted to donate above ground landscape features, that are aesthetically appropriate, do not cause any damage and are easily removable (e.g., garden bench, terra cotta or glazed planters, plant stands, sealed bottom, raised bed planters...)`. Is there a process for this? Can anyone just put anything outside or do they need permission?

`All donations can be removed at any time, without notice, if any board member assesses the item to be inappropriate in aesthetic nature or placement.` seems a bit harsh. If there's a process to get permissions to put something outside in the common area, we could keep track of what is who's. And then we can add something to this description that says something like: **If we wish to remove an item and if we know who placed the item, we will try out best to notify the resident that placed it there so that they may remove if they'd like, although there will be no guarantees that we know  who or have the means to contact the resident.**